### PR TITLE
Feat: Details Page Spacing

### PR DESCRIPTION
**Pull request by theogainey**
### Description

add constant spacing to details page 
add hover and focus states to copy bio button

### Spec

Designs: [Designs](https://www.figma.com/file/kswpPSHH2h5njVyfGDbIHk/FSA-Capstone-Project-2022)

See Story: [227](https://sparkbox.atlassian.net/browse/FSA22V1-227)

### Validation

* [ ] This PR has visual elements, so it was reviewed by a designer.
* [ ] This PR has code changes, and our linters still pass.
* [ ] This PR affects production code, so it was browser tested (see below).
* [ ] Changes are browser tested. This includes functionality, accessibility, responsiveness, and design.

#### To Validate

1. Make sure all PR Checks have passed (GitHub Actions, CircleCI, Code Climate, Snyk, etc).
2. Pull down all related branches.
3. Confirm all tests pass.
4. On multiple viewport sizes, visually inspect details page for any inconsistent spacing, layout issues, or significant deviations from Figma design.

### CODE REVIEW

**Review by empee3 — CHANGES_REQUESTED** [review]
This is looking great! One small issue I noticed is that the "Reset Filters" link isn't aligned with the right of the "Unavailable" button as it is in the design. I think if you add left and right padding of `1rem` to `.cmp-reset-filters`, you'd be good to go.

**Review by empee3 — DISMISSED** [review]
Pretty Good

File: styles/components/_fun-facts.scss [hunk 1]
```diff
@@ -1,5 +1,6 @@
 .cmp-fun-facts {
   width: 100%;
+  margin-top: space("xl");
```
Comment (arnest00) [inline styles/components/_fun-facts.scss:3]: For FunFacts, PastClients, and PersonalBio we could similarly have the largest, Figma margin here, too.
```css
  @media (min-width: $lg-screen-width) {
    margin-top: space("3xl");
  }
```

File: styles/components/_past-clients.scss [hunk 2]
```diff
@@ -1,5 +1,6 @@
 .cmp-past-clients {
   width: 100%;
+  margin-top: space("xl");
```
Comment (arnest00) [inline styles/components/_past-clients.scss:3]: For FunFacts, PastClients, and PersonalBio we could similarly have the largest, Figma margin here, too.
```css
  @media (min-width: $lg-screen-width) {
    margin-top: space("3xl");
  }
```

File: styles/components/_personal-bio.scss [hunk 3]
```diff
@@ -4,6 +4,7 @@
 
   letter-spacing: -0.02em;
   width: 100%;
+  margin-top: space("xl");
```
Comment (arnest00) [inline styles/components/_personal-bio.scss:7]: For FunFacts, PastClients, and PersonalBio we could similarly have the largest, Figma margin here, too.
```css
  @media (min-width: $lg-screen-width) {
    margin-top: space("3xl");
  }
```

File: styles/components/_button.scss [hunk 4]
```diff
@@ -1,9 +1,18 @@
...
-    text-decoration: underline;
+    @media (hover: hover) {
+      &:hover {
+        text-decoration: underline;
+      }
+    }
+
+    &:focus {
+      @include focus-styles;
+    }
```
Comment (arnest00) [inline styles/components/_button.scss:16]: This isn't the right SCSS file for this, but the Copy Bio button's lopsided.
Screen Shot 2022-06-14 at 2 06 49 PM
Adding `padding: 0;` to `.cmp-copy-to-clipboard-button` fixed this for me.

**Review by arnest00 — CHANGES_REQUESTED** [review]
Some style fixes I found. If you'd met with a designer about the margin comments I left, let me know!

**Review by arnest00 — APPROVED** [review]
This is looking beautiful.

**Pull request merged**